spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe User, type: :model do
+
+  context "validation" do
+    subject(:user) { User.new(sunet_id: "jdoe") }
+
+    it "is not valid unless it has all required attributes" do
+      expect(User.new).not_to be_valid
+      expect(user).to be_valid
+    end
+
+    it { is_expected.to validate_uniqueness_of(:sunet_id) }
+    it { is_expected.to have_db_index(:sunet_id) }
+    it { is_expected.to validate_presence_of(:sunet_id) }
+    it { is_expected.to have_db_column(:sunet_id).of_type(:string)}
```

Comment:
`have_db_*` seem overly literal to me, since they effectively test the schema, not the model.  

`validate_presence_of(:sunet_id)` is effectively what lines 7-8 already show.